internal/rollup/rollup.config.js

```diff
@@ -62,6 +63,11 @@ function resolveBazel(importee, importer, baseDir = process.cwd(), resolve = req
 
   if (!resolved) {
     // workspace import
+    const userWorkspace = `${workspaceName}/`;
```

Comment:
doesn't look Windows-friendly, would be better to use path.relative

Reply:
hmmm. maybe I need more coffee but how would check if the path starts with a `workspaceName` segment with path.relative?

Reply:
I use this
```
> let path = "mywksp/path/to/file"
undefined
> require('path').relative("mywksp", path)
'path/to/file'
> require('path').relative("otherwksp", path)
'../mywksp/path/to/file'
```

so it would be `if (!path.relative(TMPL_workspaceName, resolved).startsWith("..")`

Reply:
and it also yields the path you want when it doesn't start with '..'

Reply:
Nice. Much better.